config/checkstyle_resources_suppressions.xml

```diff
@@ -84,6 +84,10 @@
     files="[\\/]InputRegexpTrailingComment10\.java"/>
   <suppress id="noTrailingWhitespace"
     files="[\\/]InputRegexpSinglelineJavaTrailingComment10\.java"/>
+  
+  <!-- avoid trailing whitespace because it's intentional -->
```

Comment:
`avoid trailing whitespace because it's intentional` -> `Input file contains trailing whitespace to kill pitest mutation`

Reply:
It is good to be explicit about suppression, in case we question this in the future it will save someone a few minutes in digging through git blame.

Reply:
Done